src/server/VRxControl.py

```diff
@@ -2,12 +2,11 @@
 # VRx Control
 #
 
-#
-#
-
 import logging
+from monotonic import monotonic
```

Comment:
Ditch the dependency monotonic and just use `time.monotonic`. It's the exact same thing.